ConnorAndersonLarson/PlanetParty
# Feature/router

# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [x] Feature (to add router)
- [x] Refactor (to add planet images rather than icons on detail view)
- [x] Style (very minor styling)

### Are there any known bugs?


### Summary of the changes
* Add Router
* Add home-ish button-ish


### What ticket(s) do the changes resolve?
#17

Please Link Issues to this PR as necessary.
https://github.com/ConnorAndersonLarson/PlanetParty/issues/17#issue-855381014

### Notes for the Reviewer
Wrapping the planets in our Planetarium component in `<Link>`s added some annoying style side effects. Same with making the Planet Party! title a `<Link>`.


### How to Test Changes?
Confirm that click on a planet takes you to a new URL, that you can get back home by clicking on the title or on the back-ish button-ish, that actual planet images appear on the planet detail view, and that typing in a bad url (e.g. `/vulcan`) just takes you to the home page (but we can change that behavior if we want!)


### Next Iterations
